src/card/generate_card_assets.rb

```diff
@@ -14,7 +14,7 @@
   printed_cards = card_specs.map.with_index do |card_spec, index|
     image_path = File.join(tmp_dir, "#{index}.png")
 
-    generator.generate(Card::Models::MOCK_CARD_SPECIFICATION, image_path)
```

Comment:
I was accidentally still using the Card Mock specification in my last PR, now I'm using the loaded card spec